darwin/torch/transforms.py

```diff
@@ -1,24 +1,74 @@
-import PIL
+import random
+from typing import Any, Dict, Optional, Union
+
 import torch
+import torchvision.transforms as transforms
+import torchvision.transforms.functional as F
+from PIL import Image
+
+from .utils import convert_polygon_to_mask
+
+TargetKey = Union["boxes", "labels", "masks", "image_id", "area", "iscrowd"]
+TargetType = Dict[TargetKey, torch.Tensor]
 
-from darwin.torch.utils import convert_polygon_to_mask
+
+class Compose(transforms.Compose):
+    def __call__(self, image: Image, target: Optional[TargetType] = None):
+        for transform in self.transforms:
+            image, target = transform(image, target)
+        return image, target
 
 
-class Compose(object):
-    def __init__(self, transforms):
-        self.transforms = transforms
+class RandomHorizontalFlip(transforms.RandomHorizontalFlip):
```

Comment:
how about def __init__(self, p) and then not using RandomHorizontalFlip as base class?